main.go

```diff
@@ -113,15 +113,11 @@ func main() {
 		}
 	}
 
-	// Print cordova and ionic version
-	cordovaVer, err := ionic.CordovaVersion()
-	if err != nil {
+	fmt.Println()
+	if err := ionic.CordovaVersion(); err != nil {
 		failf("Failed to get cordova version, error: %s", err)
 	}
 
-	fmt.Println()
-	log.Printf("cordova version: %s", colorstring.Green(cordovaVer.String()))
```

Comment:
Was this version print unnecessary?

Reply:
It was but we changed the implementation of the cordova version reader and it does the print on its own. This line is no longer needed.

Reply:
Makes sense, thanks for the clarification!